lib/mail_room/delivery/sidekiq.rb

```diff
@@ -35,6 +35,7 @@ def deliver(message)
 
         client.lpush("queue:#{options.queue}", JSON.generate(item))
 
+        MailRoom.structured_logger.info("Message pushed onto Sidekiq queue")
```

Comment:
I wonder if the interface should take a hash as last argument, where one key should be `message:`. Thus this would be more like:

```suggestion
        MailRoom.structured_logger.info(queue: 'Sidekiq', message: 'message pushed')
```